### PR TITLE
Third_module

### PR DESCRIPTION
**Pull request by AlexTikhon**

### CODE REVIEW

File: third-module/src/loaders/controller/user.controller.mjs [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+                message: `User with id:${req.params.id} not found`
+            });
+        }
+    };
+    
+    static async createUser(req, res) {
+        const userBody = req.body;
+        const user = new User(userBody);
+
+        const id = await UserModel.create(user).then(data => data.id)
```
Comment (natallia-mikhalenak) [inline third-module/src/loaders/controller/user.controller.mjs:64]: '/service' folder is empty, but should contain the logic to work with UserModel.
Please move the logic to UserService

File: third-module/src/loaders/controller/user.controller.mjs [hunk 2]
```diff
@@ -0,0 +1,110 @@
+import { UserModel } from '../../models/user.model.mjs'
+
+class User {
```
[the remark this answers is not in the document]
Comment (AlexTikhon) [reply third-module/src/loaders/controller/user.controller.mjs:3]: Done

File: third-module/src/loaders/express.mjs [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+import { ENV_CONFIG } from '../config/env.config';
+import { router } from '../router/routes.mjs';
+// import { validation } from '../validation/user.validation.mjs';
+
+export function initExpress() {
+  const app = express();
+  app.use(express.json());
+  app.use(express.urlencoded({ extended: false }));
+  app.use(router);
+//   app.use(validation.validateRequest);
```
[the remark this answers is not in the document]
Comment (AlexTikhon) [reply third-module/src/loaders/express.mjs:12]: Fixed

File: third-module/src/models/user.model.mjs [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+    },
+    {
+      id: 102,
+      login: 'wemfk@ef.com',
+      password: 'password3',
+      age: 20,
+      isDeleted: false
+    },
+  ];
+  mockedUsers.forEach(async (value) => await UserModel.create(value));
```
[the remark this answers is not in the document]
Comment (AlexTikhon) [reply third-module/src/models/user.model.mjs:67]: Fixed

File: third-module/src/loaders/controller/user.controller.mjs (hunk 1 above)
Comment (AlexTikhon) [reply third-module/src/loaders/controller/user.controller.mjs:58]: Done, thanks

**Pull request merged**